diagnostic.md

```diff
@@ -73,7 +87,16 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST http://www.example.com/donuts/$ID
+
+let postDonuts = $.ajax ({
+    url: 'http://www.example.com/donuts/',
+    method: 'POST',
+    name: "French Cruller",
```

Comment:
The data payload should be included in an attribute called `data` with the object as the value.

```javascript
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```